v2/dtos/common/ping.go

```diff
@@ -1,14 +1,31 @@
 //
 // Copyright (C) 2020 IOTech Ltd
+// Copyright (C) 2020 Intel Corporation
 //
 // SPDX-License-Identifier: Apache-2.0
 
 package common
 
+import (
+	"time"
+
+	v2 "github.com/edgexfoundry/go-mod-core-contracts/v2"
+)
+
 // PingResponse defines the content of response content for POST Ping DTO
 // This object and its properties correspond to the Ping object in the APIv2 specification:
 // https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/PingResponse
 type PingResponse struct {
-	BaseResponse `json:",inline"`
-	Timestamp    string `json:"timestamp"`
+	Versionable `json:",inline"`
+	Timestamp   string `json:"timestamp"`
```

Comment:
we should change timestamp to int64 without format, and it would be easier to be accepted by other systems

Reply:
This way is human readable and the string is in standard format that can be parsed back to an int64. This is how the swagger has it defined. IMHO, not worth changing since this API is only used by Consul which only looks are the status code... ;-)